src/middleware/processJwt.ts

```diff
@@ -0,0 +1,15 @@
+import { expressjwt } from 'express-jwt';
+import { jwtOptions } from '../auth/jwtOptions';
+import { getLogger } from '../logger';
+import type { NextFunction, Request, Response } from 'express';
+
+const logger = getLogger(__filename);
+const expressJwtMiddleware = expressjwt(jwtOptions);
+
+const processJwt = (req: Request, res: Response, next: NextFunction): void => {
+	expressJwtMiddleware(req, res, next).catch((err: unknown) => {
+		logger.error(err);
```

Comment:
Authentication or authorization code is one of those few cases where I think we do want to know exactly what happens, whether it is what is expected to happen, and therefore test coverage is of benefit.

Reply:
This is now much safer, and now handles for both the "middleware calls next and then throws" and "middleware throws without calling next" case!

I also wrote tests that mock the express-jwt to simulate both possible situations, which was QUITE FUN....